scripts/commands/regenerateApiDocs.ts

```diff
@@ -0,0 +1,158 @@
+// This code is a Qiskit project.
+//
+// (C) Copyright IBM 2024.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+import { readdir } from "fs/promises";
+
+import yargs from "yargs/yargs";
+import { hideBin } from "yargs/helpers";
+import { $ } from "zx";
+import fs from "fs";
+
+import { Pkg } from "../lib/api/Pkg";
+import { zxMain } from "../lib/zx";
+
+interface Arguments {
+  [x: string]: unknown;
+  package?: string;
+  historical: boolean;
+}
+
+const readArgs = (): Arguments => {
+  return yargs(hideBin(process.argv))
+    .version(false)
+    .option("package", {
+      alias: "p",
+      type: "string",
+      choices: Pkg.VALID_NAMES,
+      demandOption: false,
+      description: "Which package to update",
+    })
+    .option("historical", {
+      type: "boolean",
+      default: false,
```

Comment:
I think it makes more sense for this to be true by default. It's important that we remember to apply fixes to historical API docs.

Reply:
I think `--historical` still makes more sense! You'd specify it as `--no-historical`.

Generating `--current-apis` doesn't imply you don't also generate historical. That would only work if you call it `--current-apis-only`.